src/core/model/Document.cpp

```diff
@@ -152,17 +156,23 @@ auto Document::createSaveFilename(DocumentType type, const std::string& defaultS
 
     const char* format = wildcardString.empty() ? defaultSaveName.c_str() : wildcardString.c_str();
 
+    // Todo (cpp20): use <format>
+    std::ostringstream ss;
+    ss.imbue(std::locale(""));
```

Comment:
We probably want to use the configured locale, instead of the systems/users locale, but I don't know how and if this is possible (yet).